src/main/java/application/App.java

```diff
@@ -0,0 +1,42 @@
+package application;
+import http_server.RequestParser;
+import http_server.Response;
+import http_server.Server;
+import http_server.Router;
+
+public class App {
+    static Router app;
+    static Server server;
+
+    public static void main(String args[]) {
+        app = new Router();
+        createRoutes();
+
+        server = new Server(setPortNumber(args), app);
+        server.start();
+    }
+
+    private static void createRoutes() {
+        app.get("/simple_get", (RequestParser req, Response res) -> {});
```

Comment:
I'm impressed that you designed this to work like Express! I think that the `RequestParser` object reads a bit weirdly though - `RequestParser` seems like an object that shouldn't be used at this high of a level...

Reply:
Now I see that request parser just takes a string (the request) so I understand why this is used here, but I still think it reads strangely

Reply:
I'll reword them to be `request` and `response` -- It will be clearer.

The request parser gives access to the different parts of the Request like Method, Route, Headers, and Body if available.

Reply:
Oh but i think you mean that RequestParser and Request would be 2 different objects. Request would be appropriate at this level, but RequestParser feels like an implementation

Reply:
What I'm hinting at here is that there might be another abstraction you can use - I think your RequestParser should be called RequestParser because it parses requests, but you might want to consider the responsibility of RequestParser and whether a Response abstraction is necessary.